### PR TITLE
write the duration of the 10 slowest unit tests to the console

### PR DESCRIPTION
**Pull request by zoltuz**
Just a small PR to get a better understanding of the pytest running time
With the durations flag in the config file, pytest reports the run time of the unit tests to the console, like this:

Screenshot 2021-06-15 at 08 45 14

### CODE REVIEW

**Review by WilliamIX — APPROVED** [review]

**Comment by WilliamIX** [comment]
Just as a comment, the 10 slowest tests are almost certainly the combiantorial tests and some of the reaction enumeration tests.

**Comment by zoltuz** [comment]
> Just as a comment, the 10 slowest tests are almost certainly the combiantorial tests and some of the reaction enumeration tests and the notebooks (plotting slows things down).

Yes, it was expected. I just wanted to gain some visibility maybe later on save these numbers and plot over time, so we can benchmark the new code to older versions.

**Comment by zoltuz** [comment]
> Also, something seems off with Travic CL - it isn't reporting its status.

Tl;DR: I need to reopen the PR to run it.

Travis has started a new paid plan recently (but free for opensource projects) and I had to open an account for Build a Cell. And the lack of the account prevented running Travis. I can reopen the PR again, this the only way to kick Travis into action for this one. Travis worked in my repo btw:
https://travis-ci.org/github/zoltuz/biocrnpyler/builds/774604797
